docs/src/content/docs/getting-started/index.mdx

```diff
@@ -0,0 +1,358 @@
+
+### What You'll Learn
+
+By the end of this guide, you'll understand how to:
+
+- Install and configure the Synapse SDK
+- Connect to Filecoin networks (mainnet and calibration)
+- Manage payments and allowances
+- Upload and download data with Synapse SDK
+- Use advanced features like CDN and custom providers
+
+## Prerequisites
+
+Before installing the Synapse SDK, make sure you have the following:
+
+- **[Node.js](https://nodejs.org/en) 20 or higher** installed.
+- A **supported package manager** (npm, yarn, or pnpm).
+- Added [Filecoin **Calibration**](https://chainlist.org/chain/314159) into you MetaMask.
+
+### Get Test Tokens
+
+Before you start building with Synapse SDK, you'll need to request test tokens from faucets to pay transaction fees and storage fees. For the calibration testnet:
+
+**Get tFIL tokens:**
+
+- Visit the [Filecoin Calibration Faucet](https://faucet.calibnet.chainsafe-fil.io/funds.html)
+- Enter your wallet address to receive free tFIL tokens
+- You need FIL for gas fees
+
+**Get test USDFC tokens:**
+
+- Visit the [Filecoin Calibration USDFC Faucet](https://forest-explorer.chainsafe.dev/faucet/calibnet_usdfc)
+- Enter your wallet address to receive free test USDFC tokens
+- You need USDFC for storage payments
+
+## Installation
+
+The Synapse SDK requires Node.js 20+ and can be installed via npm, yarn, or pnpm:
+
+```bash
+npm install @filoz/synapse-sdk ethers
+```
+
+Note: `ethers` v6 is a peer dependency and must be installed separately.
+
+## Quick Start Example
+
+The [`Synapse`](../api/filoz/synapse-sdk/synapse/classes/synapse/) class provides a complete, easy-to-use interface for interacting with Filecoin storage services.
+
+:::note[Keep your private key safe]
+Instead of hardcoding `privateKey` in the code, you should always consider to store and access it from `.env` files. And never commit your private key.
+:::
+
+Get started with storage in just a few lines of code.
+
+```ts twoslash
+import { Synapse, RPC_URLS, TOKENS, TIME_CONSTANTS } from "@filoz/synapse-sdk";
+import { ethers } from "ethers";
+
+/**
+ * Main function to execute the storage workflow
+ */
+async function main() {
+  try {
+    // 1. Initialize SDK with your private key and RPC endpoint
+    console.log("1) Initializing SDK...");
+    const synapse = await Synapse.create({
+      privateKey: "YOUR_PRIVATE_KEY",
+      rpcURL: RPC_URLS.calibration.websocket,
+    });
+
+    console.log("✅ SDK initialized successfully!");
+    console.log(`Connected to network: ${synapse.getNetwork()}\n`);
+
+    // 2. Check current USDFC balance
+    console.log("2) Checking wallet balance...");
+    const walletBalance = await synapse.payments.walletBalance(TOKENS.USDFC);
+    const formattedBalance = ethers.formatUnits(walletBalance, 18);
+    console.log(`Wallet USDFC balance: ${formattedBalance} USDFC\n`);
+
+    // Validate sufficient balance before proceeding
```

Comment:
How do we define "sufficient balance" -- should be both inline and maybe guidance above as a recommended start for clarity.